Lib/test/test_curses.py

```diff
@@ -1364,11 +1364,9 @@ def test_move_left(self):
         self.mock_win.reset_mock()
         self.textbox.do_command(curses.KEY_LEFT)
         self.mock_win.move.assert_called_with(1, 0)
-        self.textbox.do_command(curses.KEY_RIGHT)
-        self.mock_win.move.assert_called_with(1, 2)
         self.mock_win.reset_mock()
 
-    def test_move_left(self):
+    def test_move_right(self):
         """Test moving the cursor left."""
```

Comment:
Let's fix the docstring, and for `test_move_up` and `test_move_down` below:

```suggestion
        """Test moving the cursor right."""
```

Reply:
Done!

Reply:
Whilst we're here, let's also fix these docstrings:
```python
    def test_move_up(self):
        """Test moving the cursor left."""
        self.mock_win.reset_mock()
        self.textbox.do_command(curses.KEY_UP)
        self.mock_win.move.assert_called_with(0, 1)
        self.mock_win.reset_mock()

    def test_move_down(self):
        """Test moving the cursor left."""
        self.mock_win.reset_mock()
        self.textbox.do_command(curses.KEY_DOWN)
        self.mock_win.move.assert_called_with(2, 1)
        self.mock_win.reset_mock()
```